assets/js/script.js

```diff
@@ -0,0 +1,471 @@
+$(function() {
+
+// SCREEN BUTTONS
+  function generalStartFunctions() {
+    $gameOverlay = $('.gameOverlay')
+    $gameOverlay.css({
+      'display': 'none'
+    })
+    document.getElementsByClassName('gogo')[0].volume = 0.6;
```

Comment:
You may replace this line `$('.gogo.')[0].volume = 0.6`. Why does that works but `$('.gogo').volume` doesn't work?